src/mscorlib/shared/System/Numerics/Vector.cs

```diff
@@ -2,9 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if netcoreapp
```

Comment:
Why can't we use the existing netstandard constant here (for instance just like https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/MemoryExtensions.cs#L9).

Reply:
We discussed that here: https://github.com/dotnet/coreclr/pull/16733#discussion_r172253885